pkg/operator/ceph/csi/spec.go

```diff
@@ -58,16 +58,14 @@ const (
 	DefaultCSIPluginImage   = "quay.io/cephcsi/cephcsi:canary"
 	DefaultRegistrarImage   = "quay.io/k8scsi/csi-node-driver-registrar:v1.1.0"
 	DefaultProvisionerImage = "quay.io/k8scsi/csi-provisioner:v1.2.0"
-	DefaultAttacherImage    = "quay.io/k8scsi/csi-attacher:v1.1.0"
-	DefaultSnapshotterImage = "quay.io/k8scsi/csi-snapshotter:v1.1.1"
+	DefaultAttacherImage    = "quay.io/k8scsi/csi-attacher:v1.1.1"
+	DefaultSnapshotterImage = "quay.io/k8scsi/csi-snapshotter:v1.1.0"
```

Comment:
A couple thoughts:
- We should start the appropriate automatically by default. If there is a different version depending on the K8s version, then we can detect that and make the right decision.
- The admin can specify the desired images in operator.yaml. If they specify the images in operator.yaml, they should override our defaults.
Would you open a separate issue on this?

Reply:
csi-snapshotter 1.1.0 and 1.2.0 is having breaking changes. 
`v1.2.0` requires minimum Kube version of 1.14+

I think we can pick the image version based on the Kube version deployed, this can be done as separate PR.

@travisn some image version updating requires rbac changes also.

Reply:
@Madhu-1 You're saying the newer snapshotter image requires rbac changes? Sounds fine to just add them to common.yaml with the other rbac.